fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/patient/ChunkReader.java

```diff
@@ -205,27 +227,11 @@ public Object readItem() throws Exception {
                 pageNum = 1;
                 chunkData.setPartNum(1);
                 indexOfCurrentResourceType++;
-            }
-        }
-        if (fhirTenant == null) {
-            fhirTenant = Constants.DEFAULT_FHIR_TENANT;
-            logger.fine("readItem: Set tenant to default!");
-        }
-        if (fhirDatastoreId == null) {
-            fhirDatastoreId = Constants.DEFAULT_FHIR_TENANT;
-            logger.fine("readItem: Set DatastoreId to default!");
-        }
-        if (fhirSearchPageSize != null) {
-            try {
-                pageSize = Integer.parseInt(fhirSearchPageSize);
-                logger.fine("readItem: Set page size to " + pageSize + ".");
-            } catch (Exception e) {
-                logger.warning("readItem: Set page size to default(" + Constants.DEFAULT_SEARCH_PAGE_SIZE + ").");
+                isDoDuplicationCheck = false;
+                loadedResourceIds.clear();
```

Comment:
I know we had a similar pattern in this code before with clear. 
I see this and I think... are there unintended side effects if not properly managed.  

Maybe add a explanation of the behavior to the initial definition.

Reply:
make sense, let me add.

Reply:
done